config/config.go

```diff
@@ -606,6 +606,8 @@ type ScrapeConfig struct {
 	// More than this many buckets in a native histogram will cause the scrape to
 	// fail.
 	NativeHistogramBucketLimit uint `yaml:"native_histogram_bucket_limit,omitempty"`
+	// Larger than this schema in a native histogram will be automatically reduced.
+	NativeHistogramMaxSchema uint `yaml:"native_histogram_max_schema,omitempty"`
```

Comment:
In client_golang, we try to hide the concept of a schema from the user. We ask the user to configure a `NativeHistogramBucketFactor`. Maybe we should do the same here and call this `NatieHistogramMinBucketFactor`, with a doc comment like "If native histogram buckets grow by less than this factor from one bucket to the next, merge buckets until they grow by at least this factor."

@krajorama WDYT?